### PR TITLE
scrape.py: Add store function

### PR DESCRIPTION
**Pull request by sibasish14**
Stores the info of the user in a csv file

Closes https://github.com/palash25/CoWaPS/issues/6

### CODE REVIEW

**Comment by sibasish14** [comment]
@palash25 I think you should first merge https://github.com/palash25/CoWaPS/pull/8 .
Because he has a lot of good changes. I'll update my clone once you merge it and then add the store function in that.

File: scrape.py [hunk 1]
```diff
@@ -1,7 +1,23 @@
 import requests
 from bs4 import BeautifulSoup
-
-
+import csv
+
+'''
+This Function stores the
```
Comment (palash25) [inline scrape.py]: Move the docstrings inside their respective functions

File: scrape.py [hunk 2]
```diff
@@ -1,7 +1,23 @@
...
+	File=open("user_info.csv","a")
+	with File:
+        	writer=csv.writer(File, dialect='Dialect')
+        	writer.writerow(data)
+        	File.close()
+ 
+'''
+This function fetches the 
+info about the user
+'''
```
Comment (palash25) [inline scrape.py]: Move the docstrings inside their respective functions

**Comment by palash25** [comment]
Make the requested changes and rebase. @sibasish14

**Comment by sibasish14** [comment]
@palash25 screenshots
screenshot from 2017-12-20 00-05-08
screenshot from 2017-12-20 00-05-50

**Comment by palash25** [comment]
@sibasish14 did you rebase properly why is Github showing conflicts

**Comment by sibasish14** [comment]
@palash25 Sorry I could not rebase. It's giving some errors while rebasing.

**Comment by palash25** [comment]
@sibasish14 even I'm confused what to do just open a new PR and close this one

**Comment by sibasish14** [comment]
ok @palash25

**Pull request closed without merging**